README.md

```diff
@@ -59,30 +59,30 @@ Creating machine...
 Starting local OpenShift cluster...
 OpenShift is available at https://192.168.99.100:8443.
 
-$ oc run hello-minishift --image=gcr.io/google_containers/echoserver:1.4 --port=8080 --expose --service-overrides='{"apiVersion": "v1", "spec": {"type": "NodePort"}}'
-service "hello-minishift" created
-deploymentconfig "hello-minishift" created
+$ oc create -f hello-openshift.json
```

Comment:
@jimmidyson Sorry could not follow up this PR due to busy schedule.

I am bit curious to know the advantage of having `oc new-app` here against `oc create -f <example>.json`.

I believe to get `oc new-app`, I need to create a `<template-example>.json` and then it is two commands set up as 
- `oc create -f <template-example>.json`
- `oc new-app template-example`

WDYT ?